### DISCUSSION TITLE
Raspberry Pi Global Shutter Camera don't work correctly

### DISCUSSION THREAD

**Discussion by JohnNorth777** [Motion-Project/motionplus#152] [Q&A]
Hi all,
I am trying to run the Raspberry Pi Global Shutter camera with motionplus, but apparently there's a problem somewhere - picture is stretched:
![screenshot](https://github.com/Motion-Project/motionplus/assets/169716958/53ef174b-6925-43bd-be70-56a5bff7d39f)

My setup:
Raspberry Pi 5
Raspberry Pi Global Shutter camera
Raspberry Pi OS 12 (bookworm) (Linux 6.6.30-v8-16k+ #1761 SMP PREEMPT Thu May  2 16:57:02 BST 2024 aarch64 GNU/Linux)
MotionPlus 0.2.0

I see problem in the log ([motionplus.log](https://github.com/Motion-Project/motionplus/files/15299439/motionplus.log)):
```
May 13 21:17:36 [NTC][VID][00:ml00:MyCamera] cam_start_req: Starting.
May 13 21:17:36 [DBG][VID][00:ml00:MyCamera] cam_start_req: Plane 0 of 3 length 3481600
May 13 21:17:36 [DBG][VID][00:ml00:MyCamera] cam_start_req: Plane 1 of 3 length 870400
May 13 21:17:36 [DBG][VID][00:ml00:MyCamera] cam_start_req: Plane 2 of 3 length 870400
May 13 21:17:36 [NTC][VID][00:ml00:MyCamera] cam_start_req: Image size adjusted from 1456 x 1088 to 3200 x 1088
May 13 21:17:36 [NTC][VID][00:ml00:MyCamera] cam_start_req: Finished.
```

My configuration files:
[motionplus.conf.txt](https://github.com/Motion-Project/motionplus/files/15299421/motionplus.conf.txt)
[camera1.conf.txt](https://github.com/Motion-Project/motionplus/files/15299429/camera1.conf.txt)

Does anyone have an idea where the problem is?

With "libcamera-hello", everything is fine.

Best regards, John

**Comment by Mr-Dave** [comment]
@2konrad found the solution in #155 and I just pushed that fix to the master branch.
